gui/bar_slider.py

```diff
@@ -0,0 +1,180 @@
+from PyQt5.QtCore import Qt, QRect, QSize, pyqtSignal
+from PyQt5.QtWidgets import (QWidget, QSizePolicy)
+from PyQt5.QtGui import QPainter, QBrush, QColor, QPalette
+
+
+def _get_thumb_value(x, canvas_width, right_value):
+	return round(x / canvas_width * right_value)
+
+class BarSlider(QWidget):
+	"""BarSlider is a class which implements a slider with 2 thumbs
```

Comment:
Not sure if this is a pylint thing, but it looks weird using a multiline string to comment a single line.  Why not just use # ?

Reply:
This is a standard way to declare docstrings in Python